src/main/java/com/fasterxml/uuid/Generators.java

```diff
@@ -39,6 +39,11 @@
      * synchronization but no external file-based syncing.
      */
     protected static UUIDTimer _sharedTimer;
+
+    /**
+     * The default egress network interface.
+     */
+    protected static EthernetAddress _egressIfAddr = null;
```

Comment:
This works, although I think it'd be nice to encapsulate this in a static helper class, so it'd be loaded first time needed. Basically using class loader for handling synchronization on first active use.
(I can make that change myself if need be).